v2/contribute/decisions/session/0023-jwks-endpoint.mdx

```diff
@@ -26,7 +26,7 @@ Have already have an jwks endpoint available in the Core, but it's exposed under
 
 ## Decision Outcome
 
-We should add a new endpoint under the standard path, but keep using the existing one internally:
+We should move the JWKs endpoint in a new CDI version to a standard path:
 - This will make it easier for people to use us with standard JWT verification libs on the backend
```

Comment:
```suggestion
- This will make it easier for people to use us with standard JWT verification libs on the backend
- This will allow the flexibility of users to be able to verify access tokens issued by SuperTokens in their backend APIs without also being forced to expose the JWKs endpoint from their API layer.
```